tests/StoringGroceries.tex

```diff
@@ -32,55 +36,83 @@ \subsection{Focus}
 	\end{figure}
 \end{minipage}
 
-
+% %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% Task
+%
+% %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \subsection{Task}
 \begin{enumerate}
-	\item \textbf{Opening door:} The robot starts opening the Cupboard's door. If the robot is unable to open the door, it may ask the Referee to do it instead.
-	\item \textbf{Cupboard inspection:} The robot inspects the cupboard locating and categorizing existing groceries.
-	\item \textbf{Finding the table:} The robot turns around and locates the table.  
-	\item \textbf{Table inspection:} The robot approaches the table starts analyzing the newly bought groceries (i.e. objects).
-	\item \textbf{Moving objects:} The robot chooses which object to move first from the Table to the Cupboard, allocating similar objects all together.
+	\item \textbf{Evaluating the situation:} The robot inspects its surrounding and analyzing the best course of action. In any order, the robot has to:
+	\begin{itemize}
+		\item \textit{Inspect the cupboard} (locating and categorizing existing groceries).
+		\item \textit{Open the cupboard's door.} If the robot can't open the door, it may ask the Referee to do it.
+		\item \textit{Find the table}
+		\item \textit{Inspect the table} (analyze the newly bought groceries, i.e. objects).
+	\end{itemize}
+
+	\item \textbf{Moving objects:} The moves as many objects as possible from the Table to the Cupboard (max 5), allocating similar objects all together.
+	Stacking is allowed.
 	\begin{itemize}
 		\item Objects of the same type (i.e. identical known objects or akin alike objects) must be placed one next to the other.
 		\item If the Cupboard has no object of the same type, then objects must be grouped by category (e.g. drinks with drinks, snacks with snacks, etc)
-		\item If the Cupboard has no similar object, the robot must clearly state its decision on how to solve the problem. For instance, the robot can define a place for the newly found Category (e.g. Food was found but there is no other food in the cupboard), or group all new objects together (e.g. placing all Unknown objects together).
+		\item If the Cupboard has no similar object, the robot must clearly state its decision on how to solve the problem. For instance, the robot can start a new set in a free space for either all unknown objects or all objects sharing a particular feature (color, shape, function, etc.).
+		\item Moving two objects at a time (2-handed manipulation) is allowed.
 	\end{itemize}
 
-	\textbf{Note:} Either before or after grasping an object the robot may announce the name of the object found. 
+	\textbf{Note:} Either before or after grasping an object the robot may announce the name of the object found.
 	\item \textbf{Repeat:} This repeats until the time is up or all groceries are stored.
 \end{enumerate}
 
+
```

Comment:
Is this PDF stuff still relevant?

Reply:
I would remove it but we should keep the number of changes to a minimum. Plus, it might be useful for clarification if some teams comes up with some weird way of grouping objects (unlikely) e.g. by color, shape, etc. So in case the team comes arguing that the robot grouped all cans just because of a coincidence, you need something to check against.

Reply:
In the stress of a test, checking correct classes etc is quite busy, PDF reports are easier in that regard.

Reply:
I didn't change anything here. I think @LoyVanBeek has a point, so PDF is now **optional**. Teams will score per object moved **and correctly classified**. However, robot's reasoning might not seem straightforward to us and no NLP engine will allow us to ask the robot why did it sorted out groceries as it did. Therefore, checking via the PDF report would be really useful. I think most teams will generate it.